source/usage-examples/struct-tagging.txt

```diff
@@ -7,10 +7,15 @@ Use Struct Tags
 You can specify the way that the Go Driver converts Go
 structs to :manual:`BSON </reference/bson-types/>` by using struct tags.
 
-The following code declares a ``BlogPost`` struct, where the ``WordCount`` field
-has a struct tag that sets a custom BSON field name of ``word_count``.
-By default, the driver marshals the other fields as the lowercase of the struct
-field name:
+Example
+-------
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+The following code declares a ``BlogPost`` struct, in which the
+``WordCount`` has a struct tag that sets a custom BSON field name
+of ``word_count``. By default, the driver marshals the other fields as
+the lowercase of the struct field name:
```

Comment:
This sentence is a little long and might be easier to read if separated.
"The following code declares a struct of type BlogPost. This struct contains a struct tag that maps the "WordCount" field to the BSON field name word_count. By default, the driver marshals "